docs/source/contributing/setup.md

```diff
@@ -123,6 +123,14 @@ configuration:
 jupyterhub -f testing/jupyterhub_config.py
 ```
 
+The test configuration enables a few things to make testing easier:
+
+- use 'dummy' authentication and 'simple' spawner
+- named servers are enabled
+- listen only on localhost
+- 'admin' is an admin user, if you want to test the admin page
+- disable caching of static files
+
```

Comment:
I was about to open an issue about this before but saw this in this PR so didn't :)